docs/source/api/apollo-gateway.mdx

```diff
@@ -100,11 +100,18 @@ const gateway = new ApolloGateway({
 
 ###### `supergraphSdl`
 
-`String`
+`string | SupergraphSdlHook | SupergraphSdlObject`
 </td>
 <td>
 
```

Comment:
I'd start this description with something like:

"You provide your supergraph schema to the gateway with this option. You can provide it as a `string`, via a `SupergraphSdlHook`, or via a `SupergraphSdlObject`:"

And then bold each of the three "When `supergraphSdl` is a..." phrases up to and including the colon